subsys/net/lib/nrf_cloud/src/nrf_cloud.c

```diff
@@ -126,6 +126,33 @@ int nrf_cloud_user_associate(const struct nrf_cloud_ua_param *param)
 	return err;
 }
 
+int nrf_cloud_shadow_update(const struct nrf_cloud_sensor_data *param)
+{
+	int err;
+	struct nct_cc_data sensor_data = {
+		.opcode = NCT_CC_OPCODE_UPDATE_REQ,
+		.id = 901
```

Comment:
I think this could be defined in a macro

Reply:
Fixed.